scripts/recipe-template.txt

```diff
@@ -1,9 +1,17 @@
 cask "action-server" do
   version "#VERSION#"
-  sha256 "#HASH#"
+
```

Comment:
Might need to add line:
arch arm: "arm64", intel: "x86_64"

..here, just found the homebrew docs on this.. and they are confusing:
https://docs.brew.sh/Cask-Cookbook#handling-different-system-configurations

Reply:
I'm uncertain if that'll make a difference... in those examples it uses the `arm` as a variable for the replacement in the url, so, at first glance I don't think we need (I could be wrong though -- this will probably be clearer when we actually try to do the release -- right now it's all based on docs, but the final test is doing the actual release).